src/openms/source/KERNEL/MSSpectrum.cpp

```diff
@@ -317,6 +322,49 @@ namespace OpenMS
     return (max_intensity_it - this->begin());
   }
 
+
+  void MSSpectrum::sortByPositionPresorted(const std::vector<Chunk>& chunks)
```

Comment:
write a test for this in MSSpectrum_test.cpp

Reply:
done